mscore/scoreview.cpp

```diff
@@ -2363,7 +2364,7 @@ void ScoreView::cmd(const char* s)
         { { "up-chord" }, [](ScoreView* cv, const QByteArray&) {
                 Element* el = cv->score()->selection().element();
                 Element* oel = el;
-                if (el && (el->isNote() || el->isRest())) {
+                if (el && (el->isNote() || el->isRest() || el->isMMRest() || el->isMeasureRepeat())) {
```

Comment:
I might still recommend a utility function "isRestType()" for this.  Comes up just often enough, and I can also imagine someday other types being rolled into this.

Reply:
All right. What do you think of `isRestFamily()` vs. `isRestType()`?

Reply:
Done with `isRestFamily()` (because I feel "isRestType" semantically says the same thing as "isRest" given that the function checks the element type).